saleor/graphql/middleware.py

```diff
@@ -60,13 +43,14 @@ def get_service_account(auth_token) -> Optional[ServiceAccount]:
     return qs.first()
 
 
-def service_account_middleware(get_response):
+def service_account_middleware(next, root, info, **kwargs):
 
     service_account_auth_header = "HTTP_AUTHORIZATION"
     prefix = "bearer"
+    request = info.context
 
-    def _service_account_middleware(request):
-        if request.path == API_PATH:
+    if request.path == API_PATH:
```

Comment:
AFAIK we have only API

Reply:
What do you mean @korycins? We have one or two more views e.g. to serve digital content files.